### DISCUSSION TITLE
Webview2 sometimes does not complete download.

### DISCUSSION THREAD

**Discussion by Jiri-Novak-001** [MicrosoftEdge/WebView2Feedback#4034] [Q&A]
I have a hidden download dialog in my application. Webview2 sometimes (irregularly) does not complete the download of a small (40-70kb) Pdf file - its final renaming does not occur. The application seems to be in a waiting state, but it is not stopping nor not sleeping. The file remains in the Download folder in the form <<..guid..>>.tmp. If I close the app, this tmp file is automatically deleted. If I copy the tmp file and rename it to xx.pdf, this file is regular and the pdf reader opens it.

I tried turning off my anti virus software but it didn't help.

The problem appears in all plants all customers, on diferent webpages, so it is not a matter of one PC or webserver, but of the webview2 component.

Where can the problem be? Why won't the download finish?

**Comment by novac42** [comment]
Hi @Jiri-Novak-001 Could you please provide some more information about this? Also please take a look at #3268 to see if it helps. 
	• Framework (WinUI, Win32, WinForms, UWP, WPF, etc)
	• SDK and Runtime version.
	• Operating System

**Comment by Jiri-Novak-001** [comment]
I am adding information:

Net. Framework: 4.8, WinForms, WPF
OS: Windows 10 Pro 2H2, Widows 11 Pro 2H2
Microsoft Webwiev 2 runtime: 111.0.1661.62
All computers have latest updates

Note: Tested on about 75 computers of different parameters and different Internet connections. Often it's a slower computer or a slower internet connection, but it's not out of the question on a faster computer - just not as often.

**Comment by victorhuangwq** [comment]
Hi, I'm wondering if you are still facing this issue?
